Content.Shared/GameTicking/SharedGameTicker.cs

```diff
@@ -8,6 +7,7 @@ public abstract class SharedGameTicker : EntitySystem
     {
         // See ideally these would be pulled from the job definition or something.
         // But this is easier, and at least it isn't hardcoded.
+        //TODO: Move these, they really belong in StationJobsSystem or a cvar.
         public const string FallbackOverflowJob = "Assistant";
         public const string FallbackOverflowJobName = "assistant";
```

Comment:
any reason they cant just be moved/CVar'd now?

Reply:
`const`. Guh.